exportReport/PanelSource.java

```diff
@@ -0,0 +1,214 @@
+
+		String strBy = Constant.messages.getString("exportReport.source.by");
+		JLabel lblBy = null;
+		String ttBy = Constant.messages.getString("exportReport.source.by.tooltip");
+
+		String strFor = Constant.messages.getString("exportReport.source.for");
+		JLabel lblFor = null;
+		String ttFor = Constant.messages.getString("exportReport.source.for.tooltip");
+
+		String strScanDate = Constant.messages.getString("exportReport.source.scandate");
+		JLabel lblScanDate = null;
+		String ttScanDate = Constant.messages.getString("exportReport.source.scandate.tooltip");
+
+		String strReportDate = Constant.messages.getString("exportReport.source.reportdate");
+		JLabel lblReportDate = null;
+		String ttReportDate = Constant.messages.getString("exportReport.source.reportdate.tooltip");
+
+		String strScanVer = Constant.messages.getString("exportReport.source.scanver");
+		JLabel lblScanVer = null;
+		String ttScanVer = Constant.messages.getString("exportReport.source.scanver.tooltip");
+
+		String strReportVer = Constant.messages.getString("exportReport.source.reportver");
+		JLabel lblReportVer = null;
+		String ttReportVer = Constant.messages.getString("exportReport.source.reportver.tooltip");
+
+		String strDescription = Constant.messages.getString("exportReport.source.description");
+		JLabel lblDescription = null;
+		String ttDescription = Constant.messages.getString("exportReport.source.description.tooltip");
+
+		JPanel top = null;
+		JPanel container = null;
+		JPanel content = null;
+		SpringLayout sl = null;
+
+		top = new JPanel(new FlowLayout(FlowLayout.LEFT));
+		container = new JPanel();
+		content = new JPanel();
+
+		this.setLayout(new BorderLayout());
+		this.add(top, BorderLayout.PAGE_START);
+		lblLabel = SharedFunctions.createLabel(top, lblLabel, strLabel, SharedFunctions.getTitleFont());
+
+		int[] pad =
+		{
+				0, 0, 295, 360
+		};
+		content.setLayout(new SpringLayout());
+		sl = new SpringLayout();
+		container.setLayout(sl);
+		this.add(container, BorderLayout.CENTER);
+		sl = SharedFunctions.setupConstraints(sl, content, container, pad);
+		container.add(content);
+
+		lblTitle = SharedFunctions.createLabel(content, lblTitle, strTitle, SharedFunctions.getLabelFont());
+		txtTitle = SharedFunctions.createTextField(content, txtTitle,
+				Model.getSingleton().getSession().getSessionName().toString(), ttTitle, true,
+				extension.getTextfieldLimit());
+
+		lblBy = SharedFunctions.createLabel(content, lblBy, strBy, SharedFunctions.getLabelFont());
+		txtBy = SharedFunctions.createTextField(content, txtBy, "", ttBy, true, extension.getTextfieldLimit());
+
+		lblFor = SharedFunctions.createLabel(content, lblFor, strFor, SharedFunctions.getLabelFont());
+		txtFor = SharedFunctions.createTextField(content, txtFor, "", ttFor, true, extension.getTextfieldLimit());
+
+		String date = SharedFunctions.getCurrentTimeStamp(); // Set the Scan date as the current timestamp to give user idea of format.
```

Comment:
Is it labelled as an example? If not it might be more obvious that it's a sample if you used the start of the epoch or something....vs. users wondering why the scan date is being listed as 'now', especially if they've opened a saved session and are now generating a report (could be days/weeks/months later).